ghcide/src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -645,10 +648,10 @@ getCompletions plId ideOpts CC {allModNamesAsNS, anyQualCompls, unqualCompls, qu
   if
     -- TODO: handle multiline imports
     | "import " `T.isPrefixOf` fullLine
-      && (List.length (words (T.unpack fullLine)) >= 2)
-      && "(" `isInfixOf` T.unpack fullLine
+      && isJust (getModuleName fullLine)
+      && "(" `T.isInfixOf` fullLine
```

Comment:
You can use pattern guards here to avoid the `fromJust`:
```
    | "import " `T.isPrefixOf` fullLine
    , Just moduleName <- getModuleName fullLine
    , "(" `T.isInfixOf` fullLine
```

Reply:
I forgot about pattern guards! [Brilliant!](https://youtu.be/cEpKcBkkVMY?t=8) (Nice that it's in Haskell2010 :slightly_smiling_face: )